lib/bibliothecary/parsers/maven.rb

```diff
@@ -305,7 +305,7 @@ def self.parse_pom_manifest(file_contents, parent_properties = {}, options: {})
 
             dep_hash = {
               name: "#{groupId}:#{artifactId}",
-              requirement: version,
+              requirement: version || "*",
```

Comment:
yes! I've been thinking about just moving this to `Dependency` so we can do the fallback in one place. WDYT?

Reply:
We were thinking the same thing! I'm less familiar with how this is used, but I think defaulting to "*" makes a logical sense. I think if there's a place where we expect to definitely get a value and don't, we should raise that as error at the point where we parse it.

Reply:
after updating libraries to this, I'd say let's make libraries stop filtering nil requirements so that its model validations will fail if someone creates a nil Dependency.requirement.